.ort.yml

```diff
@@ -1,23 +1,77 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: 2021 The Elixir Team
 
+excludes:
+  paths:
+    - pattern: "lib/elixir/pages/**/*"
+      reason: "DOCUMENTATION_OF"
+      comment: "Documentation"
+    - pattern: "lib/elixir/scripts/**/*"
+      reason: "BUILD_TOOL_OF"
+      comment: "Build Tool"
+    - pattern: "lib/ex_unit/examples/**/*"
+      reason: "EXAMPLE_OF"
+      comment: "Example"
+    - pattern: "lib/*/test/**/*"
+      reason: "TEST_OF"
+      comment: "Tests"
+    - pattern: "man/*"
+      reason: "DOCUMENTATION_OF"
+      comment: "Documentation"
+    - pattern: ".github/*"
+      reason: "BUILD_TOOL_OF"
+      comment: "Documentation"
+    - pattern: ".ort/*"
+      reason: "BUILD_TOOL_OF"
+      comment: "Documentation"
+
 curations:
   license_findings:
     - path: "lib/elixir/pages/images/logo.png"
-      reason: "INCORRECT"
+      reason: "NOT_DETECTED"
       comment: "Apply Trademark Policy to Elixir Logo"
       detected_license: "NONE"
       concluded_license: "LicenseRef-elixir-trademark-policy"
 
-    - path: "lib/elixir/pages/references/library-guidelines.md"
+    - path: "LICENSES/LicenseRef-elixir-trademark-policy.txt"
       reason: "INCORRECT"
-      comment: |
-        The guide mentions multiple licenses for users to choose from.
-        It however is not licensed itself by the mentioned licenses.
+      comment: "Correct LicenseRef"
+      detected_license: "LicenseRef-scancode-proprietary-license"
+      concluded_license: "LicenseRef-elixir-trademark-policy"
+
+    - path: "VERSION"
+      reason: "NOT_DETECTED"
+      comment: "Apply Trademark Policy to VERSION file"
+      detected_license: "NONE"
       concluded_license: "Apache-2.0"
 
-    - path: "**/*"
-      reason: "INCORRECT"
-      comment: "Apply default license to all unknown files"
+    - path: "**/*.{png,ico,bin,so}"
```

Comment:
For this particular one, I would be more comfortable with listing each of them explicitly, we don't want to give the wrong license if someone drops a new file. WDYT?

Reply:
Could it be written as `"{foo/bar.png,bar/baz.ico,...}"`?

Reply:
Agreed. Looking at it closer I see that `bin` and `so` files are only in the fixtures. There we already have a rule.

The `.ico` is special (logo, tm license).

This only leaves the `png`. I^ve redefined that as `lib/elixir/pages/images/**/*.png`.

WDYT?

Reply:
> This only leaves the png. I've redefined that as lib/elixir/pages/images/**/*.png.

Yeah, happy with a rule only for them!